orangecontrib/spectroscopy/widgets/gui.py

```diff
@@ -474,3 +477,152 @@ def set_default(self, v):
     def focusInEvent(self, *e):
         self.focusIn.emit()
         return QWidget.focusInEvent(self, *e)
+
+
+class VerticalPeakLine(pg.InfiniteLine):
```

Comment:
This subclass still redefines too much (copies from pg.InfiniteLine). The first superfluous line is `self.moving = False` and it is just the first.